### PR TITLE
UI changes according to API

### PR DESCRIPTION
**Pull request by Bahaaib**
This PR contains a list of TODOs on a high priority. Please Check it out 
This PR doesn't relate to any specific issue

### CODE REVIEW

File: app/src/main/java/com/example/coinowl/MainActivity.kt [hunk 1]
```diff
@@ -19,16 +19,22 @@ import androidx.appcompat.widget.AppCompatSpinner
 import java.util.*
 
 
-// @todo #3 Un-comment old code to wire up with new UI
-// @todo #4 Remove @param testCurrencyList after wire up
 // @todo #5 Implement Card Swapping Animation
 // @todo #6 Add Chart View
+// @todo #7 Remove currency list first item "ALL"
```
Comment (ammaratef45) [inline app/src/main/java/com/example/coinowl/MainActivity.kt]: @Bahaaib `ALL` is a currency (Albanian Lek)

File: app/src/main/java/com/example/coinowl/MainActivity.kt [hunk 2]
```diff
@@ -19,16 +19,22 @@ import androidx.appcompat.widget.AppCompatSpinner
...
+// @todo #8 Add spinners default selections (eg.: USD -> EUR)
+// @todo #9 Call yesterday currency rate value from API to calc up/down %ratio [formula : (today - yesterday)/100]
+// @todo #10 Set Indicators TextViews colors according to %ratio
```
Comment (ammaratef45) [inline app/src/main/java/com/example/coinowl/MainActivity.kt]: @Bahaaib Here you mean the amount textview or a new one? how it should look like?
please describe

File: app/src/main/java/com/example/coinowl/MainActivity.kt [hunk 3]
```diff
@@ -19,16 +19,22 @@ import androidx.appcompat.widget.AppCompatSpinner
...
+// @todo #11 Remove @param main_card_sign code implementation
```
Comment (ammaratef45) [inline app/src/main/java/com/example/coinowl/MainActivity.kt]: @Bahaaib Can you describe more and tell why?

File: app/src/main/java/com/example/coinowl/MainActivity.kt (hunk 1 above)
Comment (Bahaaib) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: Really! LOL
OK! Gonna refactor it

File: app/src/main/java/com/example/coinowl/MainActivity.kt (hunk 2 above)
Comment (Bahaaib) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: It's a new TextView that displays up/down in currency value relatively with its last value
(Display the number should be calculated in the comment above)

File: app/src/main/java/com/example/coinowl/MainActivity.kt (hunk 3 above)
Comment (Bahaaib) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: It's no longer exists. Replaced by "main_first_card_currency" which I've already implemented in this PR

File: app/src/main/java/com/example/coinowl/MainActivity.kt (hunk 2 above)
Comment (ammaratef45) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: @Bahaaib please tell that in the `todo` comment

File: app/src/main/java/com/example/coinowl/MainActivity.kt (hunk 3 above)
Comment (ammaratef45) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: @Bahaaib describe that in the `todo` comment

Comment (Bahaaib) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: Everything added or removed is shown in the commit versioning history

File: app/src/main/java/com/example/coinowl/MainActivity.kt (hunk 2 above)
Comment (Bahaaib) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: Everything added or removed is shown in the commit versioning history

File: app/src/main/java/com/example/coinowl/MainActivity.kt (hunk 3 above)
Comment (ammaratef45) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: @Bahaaib excuse me, I can't understand this `todo` comment or work on it
I'm not smart so please make it more clear.

File: app/src/main/java/com/example/coinowl/MainActivity.kt [hunk 4]
```diff
@@ -19,21 +19,21 @@ import androidx.appcompat.widget.AppCompatSpinner
...
-// @todo #5 Implement Card Swapping Animation
-// @todo #6 Add Chart View
-// @todo #7 Remove currency list first item "ALL"
-// @todo #8 Add spinners default selections (eg.: USD -> EUR)
-// @todo #9 Call yesterday currency rate value from API to calc up/down %ratio [formula : (today - yesterday)/100]
-// @todo #10 Set Indicators TextViews colors according to %ratio
-// @todo #11 Remove @param main_card_sign code implementation
+// TODO: Implement Card Swapping Animation
+// TODO: Add Chart View
+// TODO: Remove currency list first item "ALL"
```
Comment (ammaratef45) [inline app/src/main/java/com/example/coinowl/MainActivity.kt]: @Bahaaib Whyyyy!!!???

Comment (Bahaaib) [reply app/src/main/java/com/example/coinowl/MainActivity.kt]: Ouch! This one is redundant

**Comment by ammaratef45** [comment]
@Bahaaib Great, now the tasks are clear and we can go forward as fast as the thunder

**Pull request merged**